Client/Frontend/Browser/Tab Tray/TabTrayController.swift

```diff
@@ -189,9 +220,28 @@ class TabTrayController: UIViewController {
         tabManager.removeTabsWithUndoToast(tabManager.tabsForCurrentMode)
         applySnapshot()
     }
+        
+    private func updateEmptyPanelState() {
+        if dataSource.snapshot().numberOfItems == 0, isTabTrayBeingSearched {
+            showEmptyPanelState()
+        } else {
+            emptyStateOverlayView.removeFromSuperview()
+        }
+    }
+    
+    private func showEmptyPanelState() {
+        if emptyStateOverlayView.superview == nil {
+            view.addSubview(emptyStateOverlayView)
+            view.bringSubviewToFront(emptyStateOverlayView)
+            emptyStateOverlayView.snp.makeConstraints { make -> Void in
```

Comment:
Do you have to specify closure arguments `make -> Void in` here?
I think this can be skipped and you access it by `$0` sign

Reply:
Fixed at https://github.com/brave/brave-ios/pull/4576/commits/7d76363d6f330760b9869479022339799ce69f33